Parsers/ASimAuthentication/ARM/AuthenticationEmpty/README.md

```diff
@@ -0,0 +1,15 @@
+# Azure Sentinel ASIM Authentication Normalization Parser
+
+This template deploys the ASIM Authentication schema parser for Azure Sentinel. The parser is a part of the Azure Sentinel Information Mode.
```

Comment:
Whats the difference between the Empty and the Generic?  Maybe we should make sure this is clear in the readme?